app/(gcforms)/[locale]/(support)/unlock-publishing/actions.ts

```diff
@@ -0,0 +1,63 @@
+"use server";
+import { auth } from "@lib/auth";
+import { createTicket } from "@lib/integration/freshdesk";
+import { logMessage } from "@lib/logger";
+
+export async function unlockPublishing({
+  managerEmail,
+  department,
+  goals,
+  language = "en",
+}: {
+  managerEmail?: string;
+  department?: string;
+  goals?: string;
+  language?: string;
+}) {
+  const session = await auth();
+  if (!session) throw new Error("No session");
+
+  //Mandatory fields
+  if (!managerEmail || !department || !goals) {
```

Comment:
See comment about the need for checking if fields are present in other server action comments.